projects/tiempoCarlaSergio/src/components/Visor.vue

```diff
@@ -2,12 +2,13 @@
   <div v-if="ciudad" >
     <div>
         <!-- Nombre Ciudad-->
-        <p> {{ ciudad.name }}</p>
+        <p> {{ ciudad.nombre }}</p>
         <!-- Temperatura-->
-        <h1 class="display-1">{{ ciudad.temperature }}</h1>
+        <h1 class="display-1">{{ parseInt(ciudad.temp) }}º</h1>
```

Comment:
Esto lo podéis resolver con un filtro: 
https://vuejs.org/v2/guide/filters.html

Reply:
Ya lo hemos arreglado